addons/l10n_es/data/template/account.fiscal.position-es_common.csv

```diff
@@ -694,3 +694,123 @@
+"","","","","","","","account_tax_template_s_iva21b","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_s_iva21isp","account_tax_template_s_iva21isp","",""
+"","","","","","","","account_tax_template_s_iva21isp","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_iva21s","",""
+"","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_s_iva10b","account_tax_template_s_iva10b","",""
+"","","","","","","","account_tax_template_s_iva10b","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_s_iva10s","account_tax_template_s_iva10s","",""
+"","","","","","","","account_tax_template_s_iva10s","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_s_iva4b","account_tax_template_s_iva4b","",""
+"","","","","","","","account_tax_template_s_iva4b","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_s_iva4s","account_tax_template_s_iva4s","",""
+"","","","","","","","account_tax_template_s_iva4s","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_p_iva21_bc","account_tax_template_p_iva21_bc","",""
+"","","","","","","","account_tax_template_p_iva21_bc","account_tax_template_p_irpfnrnue0p","",""
+"","","","","","","","account_tax_template_p_iva21_sc","account_tax_template_p_iva21_sc","",""
+"","","","","","","","account_tax_template_p_iva21_sc","account_tax_template_p_irpfnrnue0p","",""
+"","","","","","","","account_tax_template_p_iva10_bc","account_tax_template_p_iva10_bc","",""
+"","","","","","","","account_tax_template_p_iva10_bc","account_tax_template_p_irpfnrnue0p","",""
+"","","","","","","","account_tax_template_p_iva10_sc","account_tax_template_p_iva10_sc","",""
+"","","","","","","","account_tax_template_p_iva10_sc","account_tax_template_p_irpfnrnue0p","",""
+"","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva4_bc","",""
+"","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_irpfnrnue0p","",""
+"","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_iva4_sc","",""
+"","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_irpfnrnue0p","",""
+"","","","","","","","account_tax_template_s_iva0","account_tax_template_s_iva0","",""
+"","","","","","","","account_tax_template_s_iva0","account_tax_template_s_irpfnrnue0","",""
+"","","","","","","","account_tax_template_p_iva0_bc","account_tax_template_p_iva0_bc","",""
+"","","","","","","","account_tax_template_p_iva0_bc","account_tax_template_p_irpfnrnue0p","",""
```

Comment:
Indeed, there seems to be some column missing here. Look how LO renders the CSV:

![image](https://github.com/user-attachments/assets/a3c24bed-3001-47f3-96dc-3a4dca48ce20)